site/content/en/docs/manual/basics/creating_an_annotation_task.md

```diff
@@ -1,130 +1,178 @@
 ---
-title: 'Creating an annotation task'
-linkTitle: 'Creating an annotation task'
+title: 'Create annotation task'
+linkTitle: 'Create annotation task'
 weight: 2
-description: 'Instructions on how to create and configure an annotation task.'
+description: 'How to create and configure an annotation task.'
 ---
 
-Create an annotation task pressing `+` button and select `Create new task` on the tasks page or on the project page.
+To start annotating in CVAT, you need to create an annotation task and specify its parameters.
 
-![](/images/image004.jpg)
+To create a task, on the tasks page or the project page click **+** and
+select **Create new task**.
 
-> Notice that the task will be created inside the organization that you had selected at the time of creation.
-> Read more about [organizations](/docs/manual/advanced/organization/).
-
-Specify parameters of the task:
+![Create new task](/images/image004.jpg)
 
 ## Basic configuration
 
-### Name
+Use a basic configuration window to create a simple task with minimum parameters.
+
+![Basic configurator](/images/basic_confugurator.jpg)
+
+To create a basic configuration task, do the following:
+
+1. In the **Name** field, enter the name of the new task.
+
+   ![Name of task](/images/image005.jpg)
+
+2. (Optional) From the **Projects** drop-down, select a project for the new task.
+   <br>Leave this field empty if you do not want to add the task to any project.
+
+   ![Select project](/images/image193.jpg)
+
+   > **Note:** Further steps are valid if the task does not belong to the project. If the task is in
+   > project, the project's labels will be applied to the task.
+
+3. On the **Constructor** tab, click **Add label**, the label constructor menu will open:
+
+   ![Label constructor](/images/image124.jpg)
+
+4. In the **Label name** field, enter the name of the label.
```

Comment:
Maybe to insert here what label is in our tool. (Labels aka classes object classes to be annotated according to specification).. Or maybe you can suggest better description

Reply:
Added it but to the Labels Shape description